src/test/java/org/fungover/thunder/PingHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package org.fungover.thunder;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.Socket;
+import static org.junit.jupiter.api.Assertions.*;
+
+class PingHandlerTest {
+
+    @Test
+    void testIsPingRequest() throws IOException {
+        assertTrue(PingHandler.isPingRequest(new byte[]{(byte) 0xC0},1));
+    }
+
+//    @Test
```

Comment:
The code that is disabled by comments could be removed :)

Reply:
Thank you for the feedback! Since the tests did not work previously, they were commented out. We have now updated the tests accordingly.